File: EpicWorkout/EpicWorkout/EpicWorkout.Android/Fragments/ScheduleFragment.cs
using System;
using System.Collections.Generic;

using Android.Content;
using Android.Support.V4.App;
using Android.Views;
using Android.Widget;
using Android.OS;

namespace EpicWorkout.Droid.Fragments
{
    public class ScheduleFragment : Fragment
    {
        public ScheduleFragment()
        {
            RetainInstance = true;
        }

        public override View OnCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
        {
            base.OnCreateView(inflater, container, savedInstanceState);
            var view = inflater.Inflate(Resource.Layout.fragment_schedule, null);

            return view;
        }
    }
}

File: EpicWorkout(Android Studio)/app/src/main/java/com/epicworkout/fernaak/epicworkout/NavFragments/ScheduleFragment.java
package com.epicworkout.fernaak.epicworkout.NavFragments;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.epicworkout.fernaak.epicworkout.R;

public class ScheduleFragment extends Fragment
{
    public ScheduleFragment(){}
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
    {
        View view = inflater.inflate(R.layout.main_fragment_schedule, null);
        return view;
    }
}
